test/test.getch.ts

```diff
@@ -381,6 +382,21 @@ describe('🥁 configuration options', () => {
     assert.deepStrictEqual(res.config.data, body);
   });
 
+  it('should not stringify the data if it is appended by a form', async () => {
```

Comment:
To have a test you could continue installing `form-data` as a development dependency, rather than a dependency.

And then you could perhaps do, `global.FormData = require('form-data')`.